### DISCUSSION TITLE
Auto update HTTP 301 links

### DISCUSSION THREAD

**Discussion by ThijmenDam** [serhii-londar/open-source-mac-os-apps#838] [Ideas]
Hi @serhii-londar, do you know if it is possible to have the pull request link validator action (_awesome_bot_) automatically update all HTTP 301 links to their new location? I have updated about 40 links manually but that was quite a tedious task. Perhaps worth to look into this!

**Comment by serhii-londar** [comment]
@ThijmenDam I don't think awesome_bot has such a feature. We can only silence them with --allow 301 option, but not override in the file.

**Comment by dinhhieu2209-blip** [comment]
t
